packages/@glimmer/integration-tests/test/managers/helper-manager-test.ts

```diff
@@ -10,423 +10,426 @@ import {
   TestHelperManager,
 } from '../..';
 import { Arguments, Owner } from '@glimmer/interfaces';
+import { FEATURES } from '@glimmer/global-context';
 
-class HelperManagerTest extends RenderTest {
-  static suiteName = 'Helper Managers';
+if (FEATURES.DEFAULT_HELPER_MANAGER) {
```

Comment:
It seems I disabled too many tests here. I'll try to limit it to the needed tests tomorrow.